### DISCUSSION TITLE
台灣「第一顆」低軌通訊衛星明升空 中央大學貢獻最大

### DISCUSSION THREAD

**Discussion by moonnight3503** [1101-GS4719-C/Python-Community-GS4719-C#826] [General]
中央大學與鴻海科技集團鴻海研究院等單位合作發展的台灣業界第一顆實驗低軌通訊衛星，低軌通訊衛星與過去地球同步衛星相比，具有對地通訊延遲低、太空環境輻射低、發射成本低等特性，現在是目前全球都積極發展的一個趨勢。低軌衛星通訊因為種種「優勢」被世人寄予厚望，有望成為解決未來通訊需求的「萬靈丹」，不僅能降低成本花費，也能減少對地通訊的延遲，讓許多企業都躍躍欲試，目前在低軌衛星概念股上也有相對應的成長，著實是一個發展的好取向。

連結:https://tw.news.yahoo.com/news/%E5%8F%B0%E7%81%A3-%E7%AC%AC-%E9%A1%86-%E4%BD%8E%E8%BB%8C%E9%80%9A%E8%A8%8A%E8%A1%9B%E6%98%9F%E6%98%8E%E5%8D%87%E7%A9%BA-%E4%B8%AD%E5%A4%AE%E5%A4%A7%E5%AD%B8%E8%B2%A2%E7%8D%BB%E6%9C%80%E5%A4%A7-033036709.html

**Comment by bonny123865** [comment]
天啊是我們央央欸，好驕傲!!
感覺未來低軌通訊衛星會有部分取代掉同步衛星的可能!

**Comment by a45320614** [comment]
中央在太空領域的研究貢獻非常卓越，尤其是去年飛鼠號的新聞，成功發射的消息真的是振奮人心!

**Comment by Pietra1226** [comment]
低軌衛星可以讓我們比較不會因為外力而導致通訊中斷，或許在戰略上也會有很大貢獻。

**Comment by WEI-CHEN-CHUN** [comment]
目前全球都在致力於發展小型衛星，目的是低成本且開發技術較不受限

**Comment by StevenWanglolz** [comment]
中央大學的驕傲!!!

**Comment by Evan0979** [comment]
其實中央最早就是以大氣太空這一方面起家的,希望未來也能繼續為國家貢獻!

**Comment by peggy0329** [comment]
中央的太空系果然名不虛傳！！
其實從新聞中就可以發現，中央在太空、大氣有許多貢獻
希望這個貢獻可以讓社會變得更方便

**Comment by dinosaur1009** [comment]
我原本不了解原來學校在太空與大氣的領域的發展，藉由這篇分享才知道，原來可以改變我們的通訊這麼多！真的很厲害
